sections/all_map_locations.tex

```diff
@@ -8,11 +8,12 @@ \subsection*{Symbols on the Map}
 \begingroup
   \renewcommand{\arraystretch}{1.5}
   \newcolumntype{Y}{>{\centering\arraybackslash}X}
-  \begin{tabularx}{\linewidth}{Y m{0.7\linewidth}}
+  \begin{tabularx}{\linewidth}{Y m{0.8\linewidth}}
     \small
     \mbox{\textbf{I-VII}} & Field Difficulty corresponding to the level of Neutral Units. \\
-    \includesvg[height=15px]{\svgs/treasure.svg} & Roll a Treasure Die and gain the indicated bonus. \\
+	\includesvg[height=15px]{\svgs/treasure.svg} & Roll a Treasure Die and gain the indicated bonus (\includesvg[height=15px]{\svgs/resource_die.svg}, \includesvg[height=15px]{\svgs/2_treasure_die.svg}, \includesvg[height=15px]{\svgs/experience.svg}, or \includesvg[height=15px]{\svgs/artifact.svg}).\index{Treasure Die} \\
```

Comment:
As they're inline, not serving as bullet points, maybe make them smaller? Say 10px?